app-backend/tiler/src/main/scala/render/RGB.scala

```diff
@@ -0,0 +1,74 @@
+package com.azavea.rf.tiler.render
+
+import geotrellis.raster._
+import geotrellis.raster.render.Png
+
+object RGB extends (MultibandTile => Png) {
+  def apply(tile: MultibandTile): Png = {
+    val (red, green, blue) =
+      if(tile.cellType == UShortCellType) {
+        // magic numbers. Fiddled with until visually it looked ok. ¯\_(ツ)_/¯
+        val (min, max) = (4000, 15176)
+
+        def clamp(z: Int) = {
+          if(isData(z)) { if(z > max) { max } else if(z < min) { min } else { z } }
+          else { z }
+        }
+        val red = tile.band(0).convert(IntCellType).map(clamp _).normalize(min, max, 0, 255)
+        val green = tile.band(1).convert(IntCellType).map(clamp _).normalize(min, max, 0, 255)
+        val blue = tile.band(2).convert(IntCellType).map(clamp _).normalize(min, max, 0, 255)
+
+        (red, green, blue)
+      } else {
+        // Planet Labs
```

Comment:
We should make sure to eliminate unecessary/unused code -- not sure what the `else` is for here